StardewArchipelago/Locations/ShopStockModifiers/FestivalShopStockModifier.cs

```diff
@@ -90,7 +90,7 @@ private bool IsItemKnownAsACheck(ShopItemData item, out string locationName, out
 
             if (item.IsRecipe)
             {
-                var stardewItem = _stardewItemManager.GetItemByQualifiedId(item.ItemId);
+                var stardewItem = _stardewItemManager.GetItemByQualifiedId(item.ItemId) ?? _stardewItemManager.GetItemByQualifiedId($"{QualifiedItemIds.OBJECT_QUALIFIER}{item.ItemId}");

```

Comment:
You can use the method `QualifiedItemIds.QualifiedObjectId(item.ItemId)`, instead of essentially rewriting its content